### DISCUSSION TITLE
Sync with Algolia

### DISCUSSION THREAD

**Discussion by sctgraham** [supabase/supabase#5367] [Questions]
Has anyone managed to write a function + trigger to sync a table with Algolia ? This would have to support inserts, updates and deletes. 

I'm not yet too familiar with Supabase functions, so I'm wondering if anyone has figured this out before I jump into writing this.

**Comment by dalkommatt** [comment]
An official integration option would be amazing too. I've been trying to write a Supabase webhook listener in Next.js to update an Algolia index, I'll post it here if I get one working.

**Reply by anz504** [reply]
Work?

**Comment by vbuser2004** [comment]
+1 for an official integration.  

Perhaps a trigger function using the http / pg_net extensions could accomplish this without the need for a separate function?

**Comment by JohnCarterGonzalez** [comment]
+1 again for official integration. Is there a status update for this question?

**Comment by andyfelder16** [comment]
An oficial integration would be amazing!

**Comment by zentag** [comment]
Official integration please?

**Comment by rogerximenez** [comment]
Hi Folks,

There's a new Algolia <> Supabase Connector that allows you to keep your data in sync. You can read about the post here: https://supabase.com/blog/algolia-connector-for-supabase.

You can also access the connector from this link once you log in to your account: https://dashboard.algolia.com/connectors/supabase/create
